Gruntfile.js

```diff
@@ -6,6 +6,8 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-postcss');
   grunt.loadNpmTasks('grunt-contrib-connect');
   grunt.loadNpmTasks('grunt-keepalive');
+  grunt.loadNpmTasks('grunt-babel');
+  require('load-grunt-tasks')(grunt);
 
 grunt.initConfig({
   connect: {
```

Comment:
@PalleZingmark Jag har försökt få med `connect` i `$ grunt` commandot genom att ha med en task i default-arrayen men får det inte att funka. Har du någon idé på hur jag kan lösa det?

Reply:
`connect` is choking on `keepalive: true` since that won't let the rest if the defined plugins run. If you instead change that to `livereload: true` and then call the `connect:server`directly it should work as intended. 

ex. 

```javascript 
[...]
  connect: {
    server: {
      options: {
        port: 9001,
        base: 'build',
        livereload: true
      }
    }
  },
[...]
grunt.registerTask('default', ['connect:server', 'watch', 'babel']); 
```

Reply:
SWÄNGLISH! 😂

Reply:
Thanks, that works perfectly! Lol on the language 😂